rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -634,6 +634,40 @@ if should_install_app "joininbox" ; then
     fi
 fi
 
+# Install JoinMarket GUI
+if should_install_app "jm"; then
+    if [ $IS_RASPI = 1 ] || [ $IS_X86 = 1 ]; then
+        JMG_UPGRADE_URL=https://github.com/manasgandy/joinmarket-gui/archive/refs/tags/%JMG_VERSION.tar.gz
+        CURRENT=""
+        if [ -f $JMG_VERSION_FILE ]; then
+            CURRENT=$(cat $JMG_VERSION_FILE)
+        fi
+        if [ "$CURRENT" != "$JMG_VERSION" ]; then
+            # Download and build JoinMarket GUI
+            cd /home/joinmarket
+
+            # Delete previous version
+            rm -rf jmg
+
+            sudo -u joinmarket wget $JMG_UPGRADE_URL -O jmg.tar.gz
+            sudo -u joinmarket tar -xvf jmg.tar.gz
+            sudo -u joinmarket rm jm.tar.gz
+            mv jmg-* jmg
+            cd jmg
+
+            # Install dependencies
+            sudo -u joinmarket python3 -m venv jmg_venv
+            sudo -u joinmarket ./jmg_venv/bin/pip install -r requirements.txt
+
+            # Setup TLS certificates
+            sudo -u joinmarket mkdir .joinmarket/ssl
```

Comment:
There may be a way to re-use an existing cert. I'll need to think about this more.